bin/spring

```diff
@@ -4,15 +4,15 @@
 # It gets overwritten when you run the `spring binstub` command
 
 unless defined?(Spring)
-  require "rubygems"
-  require "bundler"
+  require 'rubygems'
+  require 'bundler'
 
   if match = Bundler.default_lockfile.read.match(/^GEM$.*?^    (?:  )*spring \((.*?)\)$.*?^$/m)
-    ENV["GEM_PATH"] = ([Bundler.bundle_path.to_s] + Gem.path).join(File::PATH_SEPARATOR)
-    ENV["GEM_HOME"] = ""
+    ENV['GEM_PATH'] = ([Bundler.bundle_path.to_s] + Gem.path).join(File::PATH_SEPARATOR)
+    ENV['GEM_HOME'] = ''
     Gem.paths = ENV
 
-    gem "spring", match[1]
-    require "spring/binstub"
+    gem 'spring', match[1]
```

Comment:
For example, I almost never touch anything in `/bin` unless someone tells me to. Too high probability for failure and you get really annoying errors if something gets screwy.